api/config/config_local.json

```diff
@@ -103,7 +103,9 @@
     "enableCohortBuilderV2": true,
     "enableConceptSetSearchV2": true,
     "enableReportingUploadCron": true,
-    "enableCustomRuntimes": true
+    "enableCustomRuntimes": true,
+    "bufferBillingProjectsInPerimeter": false,
```

Comment:
This is the rare case of existing=true changing to false.  I can rename this to have the standard polarity.

Reply:
+1 to inverting the naming, something like `enableLazyPerimeterAssignment`?

Reply:
done